sources/xml-descriptor.asciidoc

```diff
@@ -757,9 +757,9 @@ From Bean Validation revision 1.1 onwards, mapping authors must specify the used
 
     <xs:annotation>
         <xs:documentation><![CDATA[
-            This is the XML Schema for Bean Validation constraint mapping files.
```

Comment:
So this would need similar doing in the actual XSD files themselves.

Reply:
Ok, that's handled already in https://github.com/beanvalidation/beanvalidation-api/pull/148/.